RELEASE_NOTES

```diff
@@ -3,9 +3,70 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyconfig provides 5,093 easyconfig files, for 799 different software packages,
+The latest version of easybuild-easyconfig provides 5,580 easyconfig files, for 862 different software packages,
 34 different (compiler) toolchains, 12 software bundles and 1 meta-package.
 
+v2.7.0 (March 18th 2016)
+------------------------
+
+feature + bugfix release
+- added example easyconfig files for 63 new software packages:
+    ATSAS (#616, #2587),  astropy (#2724, #2727), attr (#2706),  BamUtil (#2654),  BBMap (#2322),  BH (#2508),  CheMPS2 (#2445),  CosmoloPy (#2723, #2727), csvkit (#2639),  Firefox (#2648),  FreeXL (#2422),  GL2PS (#2667),  Glade (#2631),  htop (#2538),  IGV (#2019),  IGVTools (#2019),  ImageMagick (#2438),  jModelTest (#2529),  KEALib (#2420),  libcerf (#2656),  libgcrypt (#2201),  libglade (#2631),  libgpg-error (#2201),  libspatialite (#2431), LittleCMS (#2438),  MAST (#2542),  MLC (#2577),  MPJ-Express (#2529),  mutil (#2201),  neon (#758),  NextClip (#2544),  npstat (#2686, #2703),  Octopus (#2643),  QuickFF (#2721), p4vasp (#2328),  PCMSolver (#2445),  PFFT (#2643),  PHYLIP (#2694),  pkgconfig (#2475, #2476),  Platypus (#2618),  pplacer (#1056),  PRINSEQ (#2437, #2444, #2585),  PyFFmpeg (#2501, #2519),  PyGObject (#2443),  PyGTK (#2443),  PyOpenGL (#2628),  pyringe (#2533),  qrupdate (#2675),  rgeos (#2635),  rpmrebuild (#2402),  shift (#2201),  SNAPE-pooled (#2688),  SNPhylo (#2701),  sratoolkit (#2715),  STAR-Fusion (#2463),  statsmodels (#2719),  StringTie (#2527),  synchronicity (#2508),  testpath (#2461),  USEARCH (#2537),  VarScan (#2464),  vsc-install (#2165),  Whoosh (#2725), xprop (#2645)
```

Comment:
where are my CRLFs?!? ;-)
(I think they got converted into spaces somewhere in that pipeline! ...)